helpers/helpers_test.go

```diff
@@ -0,0 +1,10 @@
+package helpers
+
+import "testing"
+
+func TestReverseRunes(t *testing.T) {
+	want := "olleH"
+	if got := ReverseRunes("Hello"); got != want {
+		t.Errorf("ReverseRunes() = %q, want %q", got, want)
```

Comment:
Y'a des bons framework de test en go ? En ruby on a plutot tendance à fonctionner avec un Framework qui nous donne des "expect", et en Java de mémoire il y a la même chose avec JUnit + les assert

```ruby
expect(ReverseRunes("Hello")).to eq("olleH")
```

et ça a l'avantage de gérer tout seul le formatting du message d'erreur.

Reply:
It was a choice of the Go devs to ditch expection-throwing "asserts, expects", in favor of classical code ([explanation here](https://golang.org/doc/faq#assertions)).

But yes, there is a quite popular testing framework called [testify](https://github.com/stretchr/testify). But I've never used it. Shall we vote this afternoon ?